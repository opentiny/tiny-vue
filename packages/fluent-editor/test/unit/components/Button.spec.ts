import Button from '../../../src/components/Button.vue'
import { shallowMount } from '@vue/test-utils'

describe('~/src/components/Button.vue', () => {
  test('can mount', () => {
    const wrapper = shallowMount(Button)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
