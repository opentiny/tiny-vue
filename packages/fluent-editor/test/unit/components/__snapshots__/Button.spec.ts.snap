// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`~/src/components/Button.vue can mount 1`] = `"<button class=\\"Button Button-primary\\"></button>"`;
